src/primitives/index.js

```diff
@@ -0,0 +1,36 @@
+import reactPrimitives from 'react-primitives'
+import glamorous, {ThemeProvider, withTheme} from '../index'
+
+// Create glamorous component with composition
+function createComponent(componentConstructor, tag) {
```

Comment:
Does this need to accept a `componentConstructor`? Can't this simply just reference `glamorous` as that's the only use case for this function anyway?